doc/source/serve/deploying-serve.md

```diff
@@ -15,21 +15,15 @@ This section should help you:
 
 ## Lifetime of a Ray Serve Instance
 
-Ray Serve instances run on top of Ray clusters and are started using {mod}`serve.start <ray.serve.start>`.
-Once {mod}`serve.start <ray.serve.start>` has been called, further API calls can be used to create and update the deployments that will be used to serve your Python code (including ML models).
+Ray Serve instances run on top of Ray clusters and are started using {mod}`serve.run <ray.serve.run>`.
+Once {mod}`serve.run <ray.serve.run>` has been called, serve instance will be created automatically.
```

Comment:
Why have present perfect and future tense in the same sentence. This can be short and forceful, and always true.

Once {mod}`serve.run <ray.serve.run>` is called, a serve instance is automatically created.